src/lib/Persistence/Legacy/Content/URLWildcard/Query/CriterionHandler/Type.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace Ibexa\Core\Persistence\Legacy\Content\URLWildcard\Query\CriterionHandler;
+
+use Doctrine\DBAL\ParameterType;
+use Doctrine\DBAL\Query\QueryBuilder;
+use Ibexa\Contracts\Core\Repository\Values\Content\URLWildcard\Query\Criterion;
+use Ibexa\Core\Persistence\Legacy\Content\URLWildcard\Query\CriteriaConverter;
+use Ibexa\Core\Persistence\Legacy\Content\URLWildcard\Query\CriterionHandler;
+
+final class Type implements CriterionHandler
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function accept(Criterion $criterion): bool
+    {
+        return $criterion instanceof Criterion\Type;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function handle(
+        CriteriaConverter $converter,
+        QueryBuilder $queryBuilder,
+        Criterion $criterion
+    ) {
+        /** @var \Ibexa\Contracts\Core\Repository\Values\Content\URLWildcard\Query\Criterion\Type $criterion */
+        return $queryBuilder->expr()->eq(
+            'type',
+            $queryBuilder->createNamedParameter(
+                (int)$criterion->isDirect,
```

Comment:
This is not correct, see how the value is stored by `\eZ\Publish\Core\Persistence\Legacy\Content\UrlWildcard\Gateway\DoctrineDatabase::insertUrlWildcard`